client/src/App.js

```diff
@@ -41,12 +45,20 @@ function App() {
     </AppWrapper>
   )
 
-  function handleCreatePage(newUser) {
-    setUsers([newUser, ...users])
-    history.push('usersPage')
-  }
-  function goToCreatePage() {
-    history.push('createPage')
+  function handleCreatePage(user) {
+    axios.defaults = {
+      proxy: {
+        protocol: 'http',
+        host: 'localhost',
+        port: 4000,
+      },
+    }
```

Comment:
I wonder if you still need this? 🤔 I think we added it thinking that axios had a problem, but instead it was the strict validation. Maybe remove it and check if it is needed or not.